spinalcordtoolbox/utils/shell.py

```diff
@@ -17,17 +17,26 @@
 logger = logging.getLogger(__name__)
 
 
-def display_open(file):
+def display_open(file, message="\nDone! To view results"):
```

Comment:
I'm noticing that the default message starts with a `\n` newline, but in the two new calls to the function, there's no initial newline. Is that intentional?
(I would have expected the message parameter to be without newline, and the function's f-strings to add it, but that's just me.)

Reply:
Ahh, another good catch! Thank you. :)